quarkus-extension/engine/deployment/src/main/java/org/camunda/bpm/quarkus/engine/extension/deployment/impl/CamundaEngineProcessor.java

```diff
@@ -61,7 +61,7 @@ protected FeatureBuildItem feature() {
   protected void unremovableBeans(BuildProducer<UnremovableBeanBuildItem> unremovableBeansProducer) {
     unremovableBeansProducer.produce(
         UnremovableBeanBuildItem.beanTypes(
-            DotName.createSimple(DefaultContextAssociationManager.class.getName() + "$RequestScopedAssociation")));
+            DotName.createSimple(RequestScopedAssociation.class.getName())));
```

Comment:
The `beanTypes(...)` method allows for `Class` arguments as well. So we can remove the `DotName` usage:
```suggestion
            RequestScopedAssociation.class));
```

Reply:
👍  Nice catch!